charts/redpanda/values.yaml

```diff
@@ -626,7 +629,7 @@ listeners:
         # enabled: true
         # The port used for external client connections
         port: 9094
-        # If undefined, `listeners.kafka.external.port` is used
+        # If undefined, `listeners.kafka.external.default.port` is used
```

Comment:
I also just realized this comment change doesn't make sense. This change makes the comment say:

If `listeners.kafka.external.default.port` is undefined `listeners.kafka.external.default.port` will be used.